packages/workflow/src/internals.ts

```diff
@@ -416,9 +425,56 @@ export class State {
     [
       '__enhanced_stack_trace',
       (): EnhancedStackTrace => {
-        // const stacks = this.getStackTraces();
-        // const sourceMap = this.sourceMap;
-        return { locations: [], sources: {} };
+        const sourceMap = this.sourceMap;
+        const stacks = this.getStackTraces().join('\n\n');
+        const sdkInfo: SDKInfo = { name: 'typescript', version: '' };
+        const locationsPaths: FileLocation[] = [];
+        const sourceMapRecord: Record<string, FileSlice[]> = {};
+        const fileSliceSize = 50; //Lines
```

Comment:
Can you please extract this to a top level constant?

Reply:
Just wanted to make sure whether you meant to have all three constants to be brought on the global level or just
`const fileSliceSize = 50; //Lines`

Reply:
Just this one, the others are function local